src/screens/Playlist/Playlist.tsx

```diff
@@ -26,8 +26,8 @@ function Playlist({
   },
 }: RouteComponentProps<PlaylistRouteParams>) {
   const history = useHistory();
-  const config = ConfigStore.useState((state) => state.config);
-  const accessModel = ConfigStore.useState((s) => s.accessModel);
+  const config = useConfigStore((state) => state.config);
```

Comment:
Any reason not to combine these 2 configStore calls? I doubt there's much difference in efficiency, but the code is a little more concise with a single assignment, and it will scale better if anyone needs to get a 3rd property.

Reply:
Yes, good point. We had a little debate regarding this subject in our team. I think there are 4 similar options for this, but we probably should pick one and use that primarily. We like to use the Array destructuring way.

Looking at https://github.com/pmndrs/zustand#selecting-multiple-state-slices we need to add the `shallow` comparator function so that it only updates when a selected value changes.